src/components/pages/state/state-history.js

```diff
@@ -1,54 +1,58 @@
 import React from 'react'
 import Screenshots from './screenshots'
-import Table from '../../common/table'
+import { Th, Td, Table } from '../../common/table'
 import { FormatNumber, FormatDate } from '../../utils/format'
 import stateHistoryStyle from './state-history.module.scss'
 
 export default ({ history, screenshots }) => (
   <Table>
     <thead>
       <tr>
-        <th scope="col">Date</th>
-        <th scope="col">Screenshots (ET)</th>
-        <th scope="col">New Tests</th>
-        <th scope="col">Positive</th>
-        <th scope="col">Negative</th>
-        <th scope="col">Pending</th>
-        <th scope="col">Hospitalized</th>
-        <th scope="col">Deaths</th>
-        <th scope="col">Total</th>
+        <Th scope="col" alignLeft>
+          Date
+        </Th>
+        <Th scope="col" alignLeft>
+          Screenshot
```

Comment:
Per #716, let's rename this to "Screenshots (ET)"

Reply:
Ooh, good call @kevee. In fact, we can make this even better by using the [abbreviation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr) element to provide a clearer, more accessible title. Do you think we should use `EST` instead of `ET` - seems that's the [standard](https://www.timeanddate.com/time/zones/).

`(<abbr title="Eastern Standard Time">EST</abbr>)`

This gives you a tooltip and audibly reads that back on a screen reader

![image](https://user-images.githubusercontent.com/359929/80816788-2cbe8b00-8b9e-11ea-9ec2-7721c7e5d0ea.png)

Reply:
Yes, yes, and YES

Reply:
My research on `abbr` is that it isn't widely supported based on recent testing (as in, the Screen readers (SRs) do not actually announce the `titles`:

- `abbr` isn't always announced with SRs as evidenced by [PowerMapper's tests](https://www.powermapper.com/tests/screen-readers/labelling/acronym-abbr-title/) and [Paciello Group's tests](https://thepaciellogroup.github.io/AT-browser-tests/test-files/abbr.html)
- SRs will often read it phonetically, which will [change the variation based on the word itself](https://accessibility.blog.gov.uk/2017/02/08/advice-for-creating-content-that-works-well-with-screen-readers/) (Scroll down to section about abbrs like VAT.

**Options**
- Use an `aria-label` in addition to `title`, replicating the same info. 
_This seems like the best option for this use case imho.
- Use a [glossary technique](https://stackoverflow.com/questions/45776312/how-do-screen-readers-read-abbr-tags).
_A future option for when it's acronym city_

Reply:
I vote `aria-label`

Reply:
Thanks @tatianamac! I knew there would be some sort of curveball.  I agree that `aria-label` is a good idea. So we have _both_ that and `title`, like this so you still get a tooltip with a mouse:

(`<abbr title="Eastern Standard Time" aria-label="Eastern Standard Time">EST</abbr>`)

Reply:
@kevee are there other places across that site that have timezones states like this? Might be good to do a sweep and make them all consistent.

Reply:
There are two others. I'll open a teensy PR to fix those, so don't worry about it in the context of the tables.

Reply:
Just updated with a component for all timezones and updated the tables and other places that were using "ET"